tests/cucumber/steps/index.js

```diff
@@ -54,6 +54,7 @@ if (browser) {
   } else if (browser === 'chrome') {
     require('chromedriver');
     browserHeaders = [
+      'accept-language',
```

Comment:
In CI this test is failing for chrome because `accept-language` isn't present. Did this behavior differently for you locally?

Reply:
`TEST_BROWSER=chrome make unit` is passing locally for Ubuntu 20.04.3 LTS using `v18.17.1` and `v20.5.0`

Reply:
Got it. I believe this test is too brittle for reliable use. Could you remove the `@unit.client-no-headers` line from `unit.tags` in this PR?

I will create a separate issue to fix the test (it likely should be checking that certain headers aren't present, not that all present headers must match a hardcoded list).